src/store.h

```diff
@@ -74,6 +74,10 @@ class MetadataStore {
 
   MetadataStore(const Configuration& config);
 
+  // Returns a copy of the map containing the mapping from monitored
```

Comment:
Just `Returns a copy of the mapping from ...` is good enough.

Reply:
Done.